Documentation/network/kubernetes/local-redirect-policy.rst

```diff
@@ -608,3 +612,37 @@ and exposed to user space in versions >= 5.12.
 
     In order to enable this configuration starting Cilium version 1.16.0, previously applied local redirect policies
     and policies selected backend pods need to be deleted, and re-created.
+
+Override IP
+-----------
+The local redirect policy supports overriding the IP address of the backend pod.
+This can be used to redirect traffic to a different IP address than the one assigned by Kubernetes.
+This is useful when the backend pod is running in the host network namespace where it could only listen on the specific address from loopback interface(such as GKE metadata server).
```

Comment:
Nit: missing a space before the parenthesis

```suggestion
This is useful when the backend pod is running in the host network namespace where it could only
listen on the specific address from loopback interface (such as GKE metadata server).
```